apps/yapms/src/lib/components/modals/ModalBase.svelte

```diff
@@ -30,25 +30,21 @@
 
 <dialog class="modal modal-bottom lg:modal-middle" bind:this={dialog} on:close={close}>
 	<div class="modal-box flex flex-col w-full" class:!max-w-full={fullWidth}>
-		<div class="mb-6">
-			<div class="flex gap-x-2 align-middle">
-				<slot name="icon" />
-				<h3 class="flex flex-row flex-nowrap text-2xl flex-grow">
-					{#if title !== undefined}
-						<span>{title}</span>
-					{/if}
-					<slot name="title" />
-				</h3>
-				<button title="Close Modal" class="btn btn-sm btn-circle btn-error" on:click={close}>
-					<XMark class="w-5 h-5" />
-				</button>
-			</div>
-		</div>
+		<h3 class="font-bold">
+			{title ?? ''}
+		</h3>
+
+		<button
+			class="btn btn-sm lg:btn-xs btn-circle btn-error absolute right-4 top-4 lg:top-5"
```

Comment:
Do you mind terribly if we keep the larger exit button? I know it looks goofy with the larger text, but it's also a lot easier to click.